src/contracts/repository/repository.service.ts

```diff
@@ -37,6 +38,22 @@ export class RepositoryService {
     await this.initCachedStakingRouterAbiContract(blockTag);
   }
 
+  /**
+   * Init cache for each contract or wait if it makes some error
+   */
+  public async initOrWaitCachedContracts(blockTag?: BlockTag) {
+    try {
+      const _blockTag = blockTag || {
+        blockHash: (await this.providerService.getBlock()).hash,
+      };
+      await this.initCachedContracts(_blockTag);
+    } catch (error) {
+      this.logger.error('Init contracts error. Retry', error);
+      await sleep(10_000);
```

Comment:
Maybe put a `magic` number to the file with constants

Reply:
fixed